kernel/pfring/Makefile

```diff
@@ -0,0 +1,66 @@
+#
+# Copyright (C) 2017 OpenWrt.org
```

Comment:
Don't assign it to OpenWrt unless you're affiliated, use your own name or omit it.